storage/ctab/src/main/java/org/openscience/cdk/io/MDLV2000Writer.java

```diff
@@ -1099,12 +1141,18 @@ protected static String formatMDLString(String s, int le) {
      * so a 'query file' is created if the container has aromatic bonds and this settings is true.
      */
     private void initIOSettings() {
-        forceWriteAs2DCoords = addSetting(new BooleanIOSetting("ForceWriteAs2DCoordinates", IOSetting.Importance.LOW,
+        forceWriteAs2DCoords = addSetting(new BooleanIOSetting(OptForceWriteAs2DCoordinates, IOSetting.Importance.LOW,
```

Comment:
Not that I have a problem with it, but what is the reason to rename the option? Isn't it clear anyway that it is an option? (Just thinking out load and learning from you :)

Reply:
It's not renamed, look closely. I use the variable there so downstream you can use the same one, I can never remember the options and although the whole question/answer things works well for a GUI it does not when trying to call the Java API:

![image](https://user-images.githubusercontent.com/983232/39243542-b4746f00-4885-11e8-9159-200bd0da8867.png)

```
MDLV2000Writer writer = new MDLV2000Writer();
writer.getSetting(MDLV2000Writer.OptForceWriteAs2DCoordinates)
         .setSetting("true");
// or use the magic string
writer.getSetting("ForceWriteAs2DCoordinates")
          .setSetting("true");
```

Reply:
Compare this to the "recommended" way from some tests, how can you remember that :-):

```java
       MDLV2000Writer mdlWriter = new MDLV2000Writer(writer);
        Properties     prop      = new Properties();
        prop.setProperty("ForceWriteAs2DCoordinates", "true");
        PropertiesListener listener = new PropertiesListener(prop);
        mdlWriter.addChemObjectIOListener(listener);
        mdlWriter.customizeJob();
        mdlWriter.write(molecule);
```

Reply:
Oh, crap... sorry, I indeed missed the quotes :(

BTW, that recommended way was never the only way, was it? That solution is for GUIs, where the properties come from a property dialog, which happen to often use a Properties object... for specific code, the first option is most certainly more recommended!

Reply:
(still running from meeting to meeting today (which is how come I overlooked the missing quotes), and will merge tonight, if no one else objects...)

Reply:
> BTW, that recommended way was never the only way, was it? That solution is for GUIs, where the properties come from a property dialog, which happen to often use a Properties object... for specific code, the first option is most certainly more recommended!

I believe in other readers yes, it work in the V2000 writer because we store the reference. IIRC in other readers the options are copied so you can't just set it directly using 'getSetting()'.

John

Reply:
Ah, got it... same method, different behavioral implementation... yeah, we may want to harmonize that...